build.gradle

```diff
@@ -124,8 +124,8 @@ allprojects {
                 'ReturnValueIgnored',
                 'SafeLoggingExceptionMessageFormat',
                 'StreamOfEmpty',
-                'StrictUnusedVariable',
                 'ValidateConstantMessage'
+        options.errorprone.disable 'StrictUnusedVariable'
```

Comment:
@schlosna I've disabled `StrictUnusedVariable` since it's responsible for the vast majority of compiler warning spam and we can attack it separately.